living/hebrew.md

```diff
@@ -5,21 +5,42 @@ Additional names:
 - 
 
 News:
-- 
+- https://www.ynet.co.il/
+- https://www.haaretz.co.il/
+- https://www.kan.org.il/
+- https://www.mako.co.il/
 - 
 
 Culture / History:
-- 
+- https://www.ipo.co.il/
+- https://www.eretzmuseum.org.il/
+- https://www.yaarakeydar.com/
+- https://historia.co.il/blog/
+- https://www.imj.org.il/?utm_source=google&utm_medium=cpc&utm_term=%D7%9E%D7%95%D7%96%D7%99%D7%90%D7%95%D7%9F%20%D7%99%D7%A9%D7%A8%D7%90%D7%9C&utm_campaign=g&device=c&gad_source=1&gad_campaignid=1948310760&gbraid=0AAAAACwfhz8tIFjII2t1ky5MFlGRZBrKA&gclid=Cj0KCQjwmK_CBhCEARIsAMKwcD5JgRLB73OSphBrG64bzMvHxxzhHEX8NU-wEqAeBTloBJCRE2h3z1AaAtT4EALw_wcB
```

Comment:
Please remove the tracking query parameters (`?utm_source...`) from this link, thanks!